salt/states/win_lgpo.py

```diff
@@ -250,6 +251,9 @@ def set_(name,
     for policy_section, policy_data in six.iteritems(pol_data):
         pol_id = None
         if policy_data and policy_data['output_section'] in current_policy:
+            # Make the subdict keys case insensitive
+            current_policy[policy_data['output_section']] = CaseInsensitiveDict(
```

Comment:
You could probably do this with `salt.utils.dictupdate` and avoid the import.

Reply:
The purpose is to do a case-insensitive search of the dictionary keys. I don't see how to implement that with `dictupdate`. Am I missing something?

Reply:
You'd have to implement it on top of dictupdate. It's fine if you don't want to though. Either way works.